sdk/tables/Azure.Data.Tables/src/Queryable/XmlConstants.cs

```diff
@@ -5,258 +5,16 @@ namespace Azure.Data.Tables.Queryable
 {
     internal static class XmlConstants
```

Comment:
Wow.... this is a huge amount of removal.   Figured that it may be good to just sanity check and ask if it was intended...

Reply:
Yeah, all the removed stuff was unreferenced.